pymove/_version.py

```diff
@@ -442,8 +403,7 @@ def render_pep440_old(pieces):
 
 
 def render_git_describe(pieces):
-    """
-    TAG[-DISTANCE-gHEX][-dirty].
+    """TAG[-DISTANCE-gHEX][-dirty].
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)